service/raft/group0_state_machine.cc

```diff
@@ -141,20 +141,27 @@ static future<> write_mutations_to_database(storage_proxy& proxy, gms::inet_addr
 }
 
 group0_state_machine::modules_to_reload group0_state_machine::get_modules_to_reload(const std::vector<canonical_mutation>& mutations) {
+    using sys = db::system_keyspace;
```

Comment:
No, please do not introduce unfamiliar shortcuts like this. I don't think this improves readability by much.